llvm/test/Transforms/LoopVectorize/bitcast-cse.ll

```diff
@@ -0,0 +1,71 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt %s -passes=loop-vectorize -S | FileCheck %s
+
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
```suggestion
```

Not relevant, I assume?

Reply:
The target data layout and triple are needed. Otherwise, the `bitcast` + `shufflevector` pair that is then CSE'd is not created.

Reply:
Done!